polyfill/lib/calendar.mjs

```diff
@@ -68,6 +89,26 @@ export class Calendar {
     void options;
     throw new Error('not implemented');
   }
+  timeAdd(time, duration, options, constructor) {
+    void time;
+    void duration;
+    void options;
+    void constructor;
+    throw new Error('not implemented');
+  }
+  timeSubtract(time, duration, options, constructor) {
+    void time;
+    void duration;
+    void options;
+    void constructor;
+    throw new Error('not implemented');
+  }
+  timeDifference(one, two, options) {
```

Comment:
Sorry I didn't think of this before - I think we should name this `timeUntil` since we are renaming `dateDifference` to `dateUntil` in my PR as well.